### ISSUE TITLE
Problems with using Google Fonts in protostar

### ISSUE THREAD

**Issue by dioubernardo** [joomla/joomla-cms#25972]
In the advanced tab in editing the theme I put in the font field "Open Sans Condensed" 

Joomla render
```
<link href="https://fonts.googleapis.com/css?family=Open Sans Condensed" rel="stylesheet" />
<style>
h1, h2, h3, h4, h5, h6, .site-title {
   font-family: 'Open Sans Condensed', sans-serif;
}
</style>
```
Google expect
https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300 in link
acording to https://fonts.google.com/specimen/Open+Sans+Condensed

I tried to put "Open Sans Condensed:300" to correct the URL, but joomla render

```
<link href="https://fonts.googleapis.com/css?family=Open Sans Condensed:300" rel="stylesheet" />
<style>
h1, h2, h3, h4, h5, h6, .site-title {
   font-family: 'Open Sans Condensed:300', sans-serif;
}
</style>
```

In this second case the link is correct but the font-family is wrong

**Comment by ghost** [comment]
Have you asked on [Forum](https://forum.joomla.org/) as this Repository is for coding?

**Comment by brianteeman** [comment]
![image](https://user-images.githubusercontent.com/1296369/63514210-02c9aa80-c4e0-11e9-9b88-c0c85405be64.png)

**Comment by dioubernardo** [comment]
OK, but the style will be wrong

```
h1, h2, h3, h4, h5, h6, .site-title {
		font-family: 'Open Sans Condensed:300', sans-serif;
	}
```

the correct is
font-family: 'Open Sans Condensed', sans-serif; without :300

**Comment by SharkyKZ** [comment]
Test PR #25976 please.

**Comment by ghost** [comment]
Closed as having Pull Request #25976
